RocketChat/Rocket.Chat
# feat: New timeout setting for Omnichannel Webhooks

## Proposed changes (including videos or screenshots)


Costumers were complaining about duplicated data from the Omnichannel webhooks, so I added a new setting that controls the timeouts for the webhooks, in the omnichannel->webhooks panel, the default values are 5s for http timeout, and 4x that for the retry timeout, or 20s.



## Issue(s)

OC-511
## Steps to test or reproduce


Configure and send an invalid webhook, it should take 20s to retry, instead of the usual 10s.

## Further comments
